src/mumble/Log.cpp

```diff
@@ -717,14 +738,15 @@ void Log::log(MsgType mt, const QString &console, const QString &terse, bool own
 		}
 
 		// Message notification with static sounds
-		if ((flags & Settings::LogSoundfile)) {
+		if ((flags & Settings::LogSoundfile) && !(flags & Settings::LogMessageThreshold &&  g.sh.?? > 20 )) { //need current number of members in server here?
```

Comment:
```suggestion
		int connectedUsers = 0;
		{
			QReadLocker lock(&ClientUser::c_qrwlUsers);
			connectedUsers = ClientUser::c_qmUsers.size();
		}
		if ((flags & Settings::LogSoundfile) && !(flags & Settings::LogMessageThreshold &&  connectedUsers > 20 )) {
```

I _think_ this should do the trick

Reply:
Ah! I almost got there, but I didn't use a lock. Thanks.